third-party/linshare/src/test/java/org/apache/james/linshare/client/LinshareAPITest.java

```diff
@@ -247,19 +251,32 @@ void downloadShareShouldGetUploadedSharedFile() throws Exception {
         user1LinshareAPI.share(shareRequest);
         Document sharedDoc = user2LinshareAPI.receivedShares().get(0).getDocument();
 
-        byte[] sharedFile =  linshareExtension.downloadSharedFile(USER_2, sharedDoc.getId(), sharedDoc.getName());
+        byte[] sharedFile = linshareExtension.downloadSharedFile(USER_2, sharedDoc.getId(), sharedDoc.getName());
         assertThat(sharedFile).isEqualTo(FileUtils.readFileToByteArray(user1File));
     }
 
     private File templateFile() throws Exception {
         return Files.createTempFile("linshare-api-test", ".temp").toFile();
     }
 
+    @Test
+    void uploadDocumentToUserSpaceByDelegationAccount() throws Exception {
```

Comment:
can you describe what behaviour you are checking with an explicit method name containing `Should` ?

Reply:
I will